js/lib/legacy_functions.js

```
@@ -0,0 +1,156 @@
+
+const createColorObject = (_id, color, x, y, w, h, rotate = ``, styleName = ``) =>
+    createColorObject2(_id, { x, y, w, h, rotate, styleName, background: color });
+
+const createButton = (_obj, _func) => {
+    const div = createCss2Button(_obj.id, _obj.name, _func, {
+        x: _obj.x, y: _obj.y, w: _obj.w, h: _obj.h,
+        siz: _obj?.fontsize, align: _obj?.align, animationName: _obj?.animationName,
+        backgroundColor: _obj?.normalColor,
+    });
+    div.onmouseover = () => div.style.backgroundColor = _obj?.hoverColor;
+    div.onmouseout = () => div.style.backgroundColor = _obj?.normalColor;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)